app/controllers/advice_controller.rb

```diff
@@ -1,44 +1,60 @@
 class AdviceController < ApplicationController
   include AuthorizationHelper
-
+  # If current user is TA then only current user can edit and update the advice
   def action_allowed?
     current_user_has_ta_privileges?
   end
 
+  # checks whether the advices for a question in questionnaire have valid attributes
+  # return true if the number of advices and their scores are invalid, else returns false
+  def invalid_advice_status(sorted_advice, num_advices, question)
```

Comment:
Can you look for better name for this method? [Ref1](https://stackoverflow.com/questions/3874350/naming-conventions-for-java-methods-that-return-boolean
), [Ref2](https://stackoverflow.com/questions/1263160/ruby-boolean-attribute-naming-convention-and-use
)